### DISCUSSION TITLE
cs.github.com feature request: Search in specific branch of repo

### DISCUSSION THREAD

**Discussion by trevyn** [community/community#18041] [Other Feature Feedback, Questions, & Ideas]
I'd like to narrow a search to a specific branch of a specific repo, and I don't see a way to do this. In fact, it's unclear if non-default branches are even indexed?

**Comment by look** [comment]
Hi @trevyn, this is definitely something on our radar. It's difficult at GitHub's scale so we are doing some fundamental work first. You are correct that non-default branches aren't currently indexed. There was [a previous discussion about this](https://github.com/github/feedback/discussions/8564) if you'd like to take a look.
